Mantle.xcodeproj/project.pbxproj

```diff
@@ -1294,9 +1302,11 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = CDEEABFF1D33FCBD00240A4B /* tvOS-Framework.xcconfig */;
 			buildSettings = {
+				BITCODE_GENERATION_MODE = bitcode;
 				CURRENT_PROJECT_VERSION = 1;
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
+				ENABLE_BITCODE = YES;
```

Comment:
The setting is not displayed in Xcode on tvOS targets because bitcode is required, not optional on the platform.